client/src/components/review.tsx

```diff
@@ -33,6 +33,10 @@ interface props {
 }
 
 const Review = (props: props) => {
+  const API_URL = import.meta.env.DEV
+    ? `http://localhost:${import.meta.env.VITE_SERVER_PORT || 3000}/api/v1`
+    : "";
```

Comment:
Switch to use `const API_URL = `${import.meta.env.VITE_API_SERVER_URL}/api/v1`;` instead so we can use the deployed API server. Also put it out of function.